ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -360,179 +305,118 @@ export default class ConfirmTransactionBase extends Component {
       </div>
     ) : null;
 
```

Comment:
I'm confused about something. A lot of code deleted in this file seems to be necessary for EIP1559. And some that is deleted seems necessary non-EIP1559 support. For example, on develop `inlineGasControls` (either `AdvancedGasControls` or `AdvancedGasControls`) is shown on both EIP1559 networks and non-EIP1559 networks, but here it is removed completely.

Why not just replace the `process.env.SHOW_EIP_1559_UI` checks with a check for whether the current network supports eip1559?

Reply:
I think I missed this when I first reviewed because the diff of this file was hidden

Reply:
I was under the impression that all gas editing should go through the popover.  Also, we had spoken about removing the inline controls for now, since we need to restructure the data handling.

Reply:
Okay, actually, my assessment above is not right. There are some things removed: the `inlineGasControls` functionality is removed with this PR, and so is the `primaryTotalTextOverride` and `secondaryTotalTextOverride`. We still should support these.

Reply:
I can see the difficulty in having inline gas controls here, but I think we need to continue to support their use? There removal would be a degradation of UX on non-eip1559 networks.

Reply:
@danjm is it a degradation if the functionality is still available but just in the popover? Plus on non-eip1559 networks the gas controls are available on the send stage? So for a non-1559 network the user could either use the gas pop over to edit gas on the confirmation screen or navigate back to the send screen and edit gas from there?